openapi-manager-types/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "openapi-manager-types"
```

Comment:
I'm wondering if we want to call these crates `dropshot-api-manager` and `dropshot-api-manager-crates`. Thoughts?

Reply:
We decided to do this.